### PR TITLE
add site generation

### PR DESCRIPTION
**Pull request by simanw304**
add site generation

### CODE REVIEW

**Comment by simanw304** [comment]
jenkins passed http://172.168.2.101:8080/job/ZOO-PR-Validation/484/
cc @hkvision @helenlly

File: pyzoo/docs/bigdl_pytext.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```
Comment (hkvision) [inline pyzoo/docs/bigdl_pytext.py]: Can we rename this file to `analytics_zoo_pytest.py`?

File: docs/gen_site.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        p.communicate()
+        if p.returncode != 0:
+            print err_msg
+            sys.exit(1)
+    except OSError as e:
+        print err_msg
+        print e.strerror
+        sys.exit(1)
+
+parser = argparse.ArgumentParser(description='Process ZOO docs.')
```
Comment (hkvision) [inline docs/gen_site.py]: Zoo -> Analytics Zoo

File: pyzoo/docs/doc-gen.sh [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+PYSPARK=$(find -L $SPARK_HOME -name pyspark.zip)
+if [ -z $PYSPARK ]; then
+ echo 'Cannot find pyspark.zip. Please set SPARK_HOME correctly'
+ exit 1
+fi
+
+sed -i "/sys.path.insert(0/i sys.path.insert(0, '.')\nsys.path.insert(0, u'$PYSPARK')" conf.py
+sed -i "/^extensions/s/^extensions *=/extensions +=/" conf.py
+sed -i "/^extensions/i extensions = ['sphinx.ext.autodoc', 'sphinx.ext.mathjax', 'bigdl_pytext']" conf.py
```
Comment (hkvision) [inline pyzoo/docs/doc-gen.sh]: analytics_zoo_context?

**Comment by hkvision** [comment]
Others look good. After the minor modifications, we can merge it first.

**Comment by simanw304** [comment]
jenkins passed http://172.168.2.101:8080/job/ZOO-PR-Validation/489/

**Pull request merged**